lib/tasks/evm_test_helper.rb

```diff
@@ -1,5 +1,6 @@
 module EvmTestHelper
-  VMDB_SPECS        = FileList["spec/**/*_spec.rb"].exclude(/^spec\/(replication|migrations|automation)/)
+  #TODO: Reuse the array of excluded directories for building the FileList exclusion
+  VMDB_EXCLUDED_SPEC_DIRECTORIES = %w(replication migrations automation).freeze
```

Comment:
No, we can't.  We need to provides a file list in a way that both parallel_tests and rspec understand.  parallel_tests needs to calculate the files so it can split the work and dispatch it to rspec.